src/main/resources/liquibase/crop_changelog/21_4_0.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="cuenyad" id="v21.4.0-1">
+			<preConditions onFail="MARK_RAN">
+				<and>
+					<sqlCheck expectedResult="0">select count(*) from cvterm where name = 'Female Parent' and cv_id = 1010 and is_obsolete = 0 and cvterm_id = 8341;</sqlCheck>
```

Comment:
@cuenyad cvterm_id check should be done in a different sqlCheck because it is the key of the table

Reply:
done!